src/index.js

```diff
@@ -1,63 +1,48 @@
 import './style.css';
+import createTodo from '../modules/create.js';
+// import NewTodo from '../modules/todoClass.js';
```

Comment:
- [ ] Kindly remove commeted code as this makes your codebase cleaner and more readable

Reply:
![secondProject](https://user-images.githubusercontent.com/92423023/214769371-201fd07f-7a11-4204-ad1f-3b2d184ca4ce.jpg)